src/components/ActivityPage/Transaction/Transaction.tsx

```diff
@@ -280,7 +280,7 @@ const Transaction = (props: Props) => {
         <TransactionDetail
           address={address}
           text={
-            isAllowed(allowance) ? (
+            isEnoughClaimMana(allowance) ? (
```

Comment:
Based on the [comment below](https://github.com/decentraland/builder/pull/1224/commits/44fddeb119dfc359c9026a016a1f3110785a558b#r547029705), we should change this check against `0` too.